pkg/config/remote/service/service.go

```diff
@@ -632,6 +633,16 @@ func (s *CoreAgentService) calculateRefreshInterval() time.Duration {
 
 func (s *CoreAgentService) refresh() error {
 	s.Lock()
+
+	// We can't let the backend process an update twice in the same second due to the fact that we
+	// use the epoch as the version for the TUF Director Targets. If this happens, the update will
+	// appear to TUF as being identical to the previous update and it will be dropped.
```

Comment:
```suggestion
	// We can't let the backend process an update twice in the same second due to the fact that we
	// use the epoch with seconds resolution as the version for the TUF Director Targets. If this happens,
	// the update will appear to TUF as being identical to the previous update and it will be dropped.
```